department-of-veterans-affairs/va.gov-cms
# VACMS-5180: Update email links and location info display.

## Description
See #5180

## Testing done
Visual / behat

## Screenshots
![image](https://user-images.githubusercontent.com/2404547/122140255-cf74b000-ce18-11eb-978e-3396c58ae5d6.png)

## QA steps
- As an administrator, go to `/node/1322/edit` and visually verify that the location info now appears in a tooltip'd fieldset, and hovering over the tooltip icon reveals fieldset info.
- Click the `contact an administrator` link, and visually verify it follows the template guidelines defined by Rachel in ticket.
- Go to view page for node 1322, click the `contact an administrator` link, and visually verify it follows the template guidelines defined by Rachel in ticket.
- Go to view page for `node/3774/edit`, click the `contact an administrator` link, and visually verify it follows the template guidelines defined by Rachel in ticket.
- Go to view page for node 3774, click the `contact an administrator` link, and visually verify it follows the template guidelines defined by Rachel in ticket.

### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
